roles/backend_setup/tasks/main.yml

```diff
@@ -8,15 +8,22 @@
 # https://wiki.gentoo.org/wiki/LVM/en
 # https://serverfault.com/questions/981694/why-does-xfs-uses-lvm-cache-chunk-size-instead-the-raid5-setup-for-sunit-swidth
 
-- name: Check if vdsm-python package is installed or not
-  command: rpm -q vdsm-python
-  register: rpm_check
-  ignore_errors: yes
+
+# Using Ansible facts to detect vdsm-python package is installed
+- name: Gather the package facts
+  ansible.builtin.package_facts:
+    manager: auto
+
+- name: Print the package facts
```

Comment:
I'm not clear why you need to do this? (debug leftovers?)

Reply:
The debug will be visible only with '-v' or more , but won't be triggered with regular run.
That's due to 'verbosity: 1'.

Reply:
It shouldn't be a problem at all and it won't slow the code.

Reply:
And future debugging is easierwithout changing the code.